<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>807</width>
    <height>590</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>531</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_A">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>290</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_D">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>290</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_R">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>290</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>R</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_D">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>22</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_R">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>22</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="sliderPosition">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_A">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>22</width>
      <height>251</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_Aval">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Dval">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Rval">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_SL">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>30</y>
      <width>22</width>
      <height>251</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_SL">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>290</y>
      <width>21</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>18</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>SL</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_SLval">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_loC">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_D">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Db">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>360</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_B">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_E">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Eb">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>360</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_G">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Gb">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>360</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_F">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Ab">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>360</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_Bb">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>360</y>
      <width>31</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_A">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QToolButton" name="toolButton_hiC">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>360</y>
      <width>31</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSlider_volume">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>370</y>
      <width>22</width>
      <height>141</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>520</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Volume</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_volume">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>350</y>
      <width>59</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Stop">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>360</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_quit">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>390</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Quit</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>340</y>
      <width>131</width>
      <height>32</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>Start Note 24</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start Note 36</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start Note 48</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start Note 60</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start Note 72</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start Note 84</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Start Note 96</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>807</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
